drizzlepac/hlautils/diagnostic_json_harvester.py

```diff
@@ -14,33 +18,160 @@
 import drizzlepac.hlautils.diagnostic_utils as du
 from stsci.tools import logutil
 
-
 __taskname__ = 'diagnostic_json_harvester'
 
 MSG_DATEFMT = '%Y%j%H%M%S'
 SPLUNK_MSG_FORMAT = '%(asctime)s %(levelname)s src=%(name)s- %(message)s'
 log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout,
                             format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
 
+
+# ------------------------------------------------------------------------------------------------------------
+
+
+def get_json_files(search_path="", log_level=logutil.logging.INFO):
+    """use grep to create a list of json files to harvest
+
+    Parameters
+    ----------
+    search_path : str, optional
+        directory path to search for .json files. Default value is the current working directory.
+
+    log_level : int, optional
+        The desired level of verboseness in the log statements displayed on the screen and written to the
+        .log file. Default value is 'INFO'.
+
+    Returns
+    -------
+    sorted_json_list : list
+        list of json files to harvest, sorted first alphabetically by filename
+    """
+
+    # set up search string and use glob to get list of files
+    log.setLevel(log_level)
+    if search_path != "" and not search_path.endswith("/"):
+        search_path += "/"
```

Comment:
It would be more Pythonic to replace the '/' with os.sep.  Also, os.join in the next line may eliminate the need for this altogether(??); at least, it would be something to look into since use of an explicit directory name separator prevents this code from working under Windows (I am trying to get my Windows environment set up to build this code now...).

Reply:
Issue resolved in commit [602ed83](https://github.com/spacetelescope/drizzlepac/pull/656/commits/602ed83400949be5716487865cc3022d80d99358)